lesson04.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<p>I know I said no math but what do you think of adding in the \sigma_{11} \sigma_{12} symbols here? This might make it easier for folks to read other sources and correlate the english with the math</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>